js-exercises/deep-copy-object/deepCopyObject.js

```diff
@@ -0,0 +1,22 @@
+const deepCopyObject = objToCopy => {
+  if (typeof objToCopy != "object" || objToCopy === null) return objToCopy;
+
+  let clone = {};
+  if (Array.isArray(objToCopy)) {
+    clone = [];
+  }
+
+  for (var i in objToCopy) {
+    if (objToCopy[i] != null && typeof (objToCopy[i]) == "object")
```

Comment:
Consider exporting this operation to a function and call it 'isObject' which returns a boolean.